app-drawer/app-drawer.html

```diff
@@ -246,6 +265,8 @@
         // Only transition the drawer shortly after it is attached (e.g. app-drawer-layout
         // may need to set the initial opened state which should not be transitioned).
         this.$.contentContainer.style.transitionDuration = '0s';
+
+        document.addEventListener('focus', this._focusHandler.bind(this), true /* useCapture */);
```

Comment:
We should add the focus listener when the drawer is opened and remove it when the drawer is closed.  We don't want all focus events coming to the `app-drawer`'s handler all the time if they don't need to.